src/module/W2lModule.cpp

```diff
@@ -417,6 +417,13 @@ std::shared_ptr<Module> parseLines(
         std::stoi(params[6]));
   }
 
+  /* ========== Precision Cast  ========== */
+  if (params[0] == "PC") {
+    LOG_IF(FATAL, params.size() != 2) << "Failed parsing - " << line;
+    auto targetType = fl::stringToAfType(params[1]);
```

Comment:
What's the intended behavior here if the user didn't specify `--mixedprecision` when they start training?

It might be reasonable to throw an exception if we're trying to add this module but the user has omitted the flag (or it wasn't included amongst the model's serialized flags). That way, the user can unambiguously know when they're dealing with an fp16 model and won't use fp16 components unless they explicitly want to or the model they're continuing/forking was already trained with them.

Reply:
Great Point. Fixed.